src/components/theme_color_scheme/html/component.hbs

```diff
@@ -0,0 +1,23 @@
+{{#with component.data}}
+  <div class="qld__theme_color_scheme" data-component="qgds-theme_color_scheme">
+    <button class="theme-toggle" id="theme-toggle" title="Toggles light & dark" aria-label="auto" aria-live="polite">
+      <svg class="sun-and-moon" aria-hidden="true" width="24" height="24" viewBox="0 0 24 24">
+        <mask class="moon" id="moon-mask">
```

Comment:
very clean. if you wish, you can externalize these svg's and include them on 'build' 
i.e.
place the file next to the hbs file and use the markup ``{{ embedSvgs "./sun-and-moon.svg" }}``

then https://github.com/qld-gov-au/qgds-vanilla/blob/main/.esbuild/plugins/handlebarsEmbedSvgPlugin.js will inline it on build as well as handle this via storybook. (do give it a try)